src/streamlink/plugins/tv999.py

```diff
@@ -0,0 +1,40 @@
+import re
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import validate
+from streamlink.stream import HLSStream
+
+
+class TV999(Plugin):
+    url_re = re.compile(r'https?://(?:www\.)?tv999\.bg/live\.html')
+    iframe_re = re.compile(r'<iframe.*src="([^"]+)"')
+    hls_re = re.compile(r'src="([^"]+)"\s+type="application/x-mpegURL"')
```

Comment:
Just saying this in general, and it isn't strictly meant in regards to your changes, but maybe we should stop using those wonky regexes for extracting data from HTML nodes and instead use a proper XML/HTML parser to get the nodes via xpath queries or CSS selectors and then read the attribute values. But ignore this now, just wanted to mention it.